client/src/AppHeader/AppHeader.scss

```diff
@@ -6,6 +6,13 @@
   text-align: center;
   background-size: cover;
 
+  .add-restaurant {
+    position: absolute;
+    right: 35px;
+    top: 24px;
+    cursor: pointer;
+  }
+
```

Comment:
not related to those lines, but gtk:
`flexbox`, make sure you cover it, it will help you later